### DISCUSSION TITLE
customizing login views, when incorrect password specified

### DISCUSSION THREAD

**Discussion by benkoshy** [janko/rodauth-rails#165] [Ideas]
Not a bug, but consider the following steps: 

1. Navigate to Login page
2. Type in the incorrect password.
3. The following appears (see pic below).

![image](https://user-images.githubusercontent.com/15097447/193702099-11d35807-b2bb-44d7-bbbb-bde9fc9bde62.png)

![Uploading image.png…]()

**Question:** is there any way of adding or displaying an "OR" to communicate options clearly to users? 

The result seems to be a conflation of three views: 

1. Reset password
2. Normal login 
3. login via sign up.

my two cents

**Comment by janko** [comment]
I if understood correctly, this is more of a design question, and I'm not a good designer 😛. I don't think default view templates need to be perfect in terms of design, it's enough to achieve the level of Bootstrap templates. There it's at least a bit more obvious that the email auth button is an alternative, given that it's right under the login button, and they're both primary buttons.

<img width="500" alt="Screenshot 2022-10-04 at 10 33 31" src="https://user-images.githubusercontent.com/795488/193773020-387c6a5d-fecb-43d6-ad42-76fec0b65537.png">
